src/main/java/com/gam/api/service/user/UserServiceImpl.java

```diff
@@ -191,6 +186,23 @@ public WorkPortfolioGetResponseDTO getPortfolio(Long requestUserId, Long userId)
         return WorkPortfolioGetResponseDTO.of(isScraped, works);
     }
 
+    @Override
+    public List<UserDiscoveryResponseDTO> getDiscoveryUsers(Long userId) {
+        val users = userRepository.findAllByIdNotOrderBySelectedFirstAtDesc(userId);
+
+        return users.stream().map((user) -> {
+            val targetUserId = user.getId();
+            val firstWorkId = user.getFirstWorkId();
+            val firstWork = findWork(firstWorkId);
+
+            val userScrap = userScrapRepository.findByUser_idAndTargetId(userId, targetUserId);
+            if (Objects.nonNull(userScrap)){
```

Comment:
```suggestion
            if (Objects.isNull(userScrap)) {
                return UserDiscoveryResponseDTO.of(user, false, firstWork);
            }
            return UserDiscoveryResponseDTO.of(user, userScrap.isStatus(), firstWork);
```

이건 어떨까요? 널일때 한가지고, not null일때 두가지 경우(있는데 false, true)라서 한가지인 경우를 먼저 걷어내는게 좋다 생각해서요! 이건 동의하지 않으면 넘어가도 됩니다. 

다만 들여쓰기 위치는 잘 맞춰주세요!

Reply:
좋은것같습니다!